### PR TITLE
Add detection benchmark for pose models

### PR DESCRIPTION
**Pull request by zoooo0820**
this PR is based on [#1107](https://github.com/PaddlePaddle/benchmark/pull/1107)

1. Add two pose model benchmark : hrnet(pose) and higherhrnet.
2. Add solov2 benchmark.

Due to the usage of frameworks are a little bit different, the .sh scripts were modified to adapt  different frameworks.

### CODE REVIEW

File: OtherFrame/detection/PyTorch/run_PyTorch.sh [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        sleep 60;
+        CUDA_VISIBLE_DEVICES=0 bash run_benchmark.sh sp 8 fp32 1 gfl;
+        sleep 60;
+        CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 bash run_benchmark.sh mp 8 fp32 1 gfl;
+        sleep 60;
+
+        cd ${BENCHMARK_ROOT}/models/mmpose;
+        cp ${BENCHMARK_ROOT}/scripts/run_benchmark.sh ./;
+        cp ${BENCHMARK_ROOT}/scripts/analysis_log.py ./;
+        pip3.7 uninstall -y mmcv mmcv-full
```
Comment (mmglove) [inline OtherFrame/detection/PyTorch/run_PyTorch.sh:44]: 请直接使用pip  python，并检查其他脚本中该写法

Comment (zoooo0820) [reply OtherFrame/detection/PyTorch/run_PyTorch.sh]: done

File: OtherFrame/detection/PyTorch/run_PyTorch.sh [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+        sleep 60;
+
+        cd ${BENCHMARK_ROOT}/models/mmpose;
+        cp ${BENCHMARK_ROOT}/scripts/run_benchmark.sh ./;
+        cp ${BENCHMARK_ROOT}/scripts/analysis_log.py ./;
+        pip uninstall -y mmcv mmcv-full
+        pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.9.1/index.html
+        pip install -r requirments.txt
+        pip install -v -e .
+        CUDA_VISIBLE_DEVICES=0 bash run_benchmark.sh sp 64 fp32 10 hrnet;
```
Comment (mmglove) [inline OtherFrame/detection/PyTorch/run_PyTorch.sh:48]: 模型名字请按照表格中的详细，避免后期模型重名无法区分填写

Comment (zoooo0820) [reply OtherFrame/detection/PyTorch/run_PyTorch.sh]: done

**Review by AshburnLee — COMMENTED** [review]
重点看了下ips计算方式，没问题

File: OtherFrame/detection/PyTorch/README.md [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+```
+脚本内容,如:
+```bash
+#!/usr/bin/env bash
+
+ImageName="registry.baidubce.com/paddlepaddle/paddle:2.1.2-gpu-cuda10.2-cudnn7";
+docker pull ${ImageName}
+export BENCHMARK_ROOT=/workspace # 对应实际地址 benchmark/OtherFrameworks/video/PyTorch
+
+run_cmd="cd ${BENCHMARK_ROOT}
```
[the remark this answers is not in the document]
Comment (zoooo0820) [reply OtherFrame/detection/PyTorch/README.md:41]: done

File: OtherFrame/detection/PyTorch/README.md [hunk 4]
```diff
@@ -0,0 +1,144 @@
...
+        sleep 60;
+        CUDA_VISIBLE_DEVICES=0,1,2,3,4,5,6,7 bash run_benchmark.sh mp 8 fp32 1 gfl;
+        sleep 60;
+
+        cd ${BENCHMARK_ROOT}/models/mmpose;
+        cp ${BENCHMARK_ROOT}/scripts/run_benchmark.sh ./;
+        cp ${BENCHMARK_ROOT}/scripts/analysis_log.py ./;
+        pip uninstall -y mmcv mmcv-full
+        pip install mmcv-full -f https://download.openmmlab.com/mmcv/dist/cu102/torch1.9.1/index.html
+        pip install -r requirments.txt
```
Comment (AshburnLee) [inline OtherFrame/detection/PyTorch/README.md:80]: requirments.txt -> requirements.txt
下同

File: OtherFrame/detection/PyTorch/README.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+        sleep 60;
+
+        cd ${BENCHMARK_ROOT}/models/SOLO;
+        pip uninstall -y mmdet mmcv mmcv-full
+        pip install mmcv
+        pip install -r requirments.txt
+        pip install -v -e .
+        cp ${BENCHMARK_ROOT}/scripts/run_benchmark.sh ./;
+        cp ${BENCHMARK_ROOT}/scripts/analysis_log.py ./;
+        CUDA_VISIBLE_DEVICES=0 bash run_benchmark.sh sp 2 fp32 5 solov2;
```
Comment (AshburnLee) [inline OtherFrame/detection/PyTorch/README.md:107]: 另外solov2 默认5个epoch？这样跑会会超过15min，使得_speed 文件不能生成。

Comment (zoooo0820) [reply OtherFrame/detection/PyTorch/README.md]: down，已将 hrnet epoch改为10， solov2改为epoch 1

**Review by AshburnLee — COMMENTED** [review]
如此修改后，竞品单卡ips值，与作者提供的可以对齐
|HRNet-W32|作者测得|性能组测得|
|---|---|---|
|bs64|118.52|117.32|
|bd160|132.23|133.67|

LGTM

**Review by hysunflower — APPROVED** [review]

**Pull request merged**